CommandLine/protos/resources/GameInformation.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto2";
+package buffers.resources;
+
+import "options.proto";
+
+message GameInformation {
+  optional int32 background_color = 1;
+  optional bool embed_game_window = 2;
+  optional string form_caption = 3;
+
+  optional int32 left = 4;

```

Comment:
Points for the most awkward ordering I can think of.

Reply:
pretty sure i copied direct from xml so you can thank yoyo

Reply:
I did this, and I believe I copied it from the old backend. This is actually deprecated in GMX, only the rtf file itself is included, and YoYo only kept it for making notes, so you cannot in anyway show it at runtime. In GMS2 they created a dedicated Notes resource group which is just text files. So what ordering would you like it to have?